integration/spec_helper.rb

```diff
@@ -0,0 +1,31 @@
+# encoding: utf-8
+require_relative "supports/logstash_test_helpers"
```

Comment:
+1 on ```integration_test_helpers```

re ```supports```: I understand it as plural, I think is better to use singular names, in this case ```support```. You know stupid naming :+1: